x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -45,22 +64,27 @@ export const RulesTableHeader = ({
   selectedRulesCount,
   searchValue,
   isSearching,
+  lastModified,
 }: RulesTableToolbarProps) => (
-  <EuiFlexGroup alignItems="center" justifyContent="spaceBetween" responsive={false} wrap>
-    <Counters total={totalRulesCount} selected={selectedRulesCount} />
-    <SelectAllToggle
-      isSelectAll={selectedRulesCount === totalRulesCount}
-      clear={clearSelection}
-      select={selectAll}
-    />
-    <BulkMenu
-      bulkEnable={bulkEnable}
-      bulkDisable={bulkDisable}
-      selectedRulesCount={selectedRulesCount}
-    />
-    <RefreshButton onClick={refresh} />
-    <SearchField isSearching={isSearching} searchValue={searchValue} search={search} />
-  </EuiFlexGroup>
+  <div>
+    {lastModified && <LastModificationLabel lastModified={lastModified} />}
```

Comment:
this is admittedly rather defensive, mostly because this value comes from the first result so we need to account for cases where we don't have it yet/at all: 
1 - before we have data ("loading" of initial fetch)
2 - when we got data but results are empty 

going to change this tho so it's part of `RulesPageData` to make it clearer

Reply:
so what I was thinking is:
1 - loading state probably doesn't show the table
2 - I think the result can never be empty since those are static rules made by us

any case, it's not urgent of course, as long as that's on purpose then all good.